jsbi.mjs

```diff
@@ -458,6 +458,70 @@ class JSBI extends Array {
     }
   }
 
+  // DataView polyfills
+  static __makeDataViewSetter(funcName, viewFuncName) {
```

Comment:
It won't hurt to just write 4 simple and straight forward static functions for this polyfill. It will make code more self-explanatory with even fewer comments.

Moreover, the current code is almost impossible for the bundler to perform dead code removal if the functions are not used.

Reply:
So you suggest duplicating both functions (making it 4 instead of 2)? I guess it doesn't hurt, but IMO will make it harder to maintain.